src/common/SecureStore.ts

```diff
@@ -35,9 +35,9 @@ export class SecureStore<Stored> {
    * Stores an object as { username, password } at key com.lifeomic.securestore/<instanceIdentifier>
    * @param object
    */
-  async setObject(itemId: string, object: Stored) {
```

Comment:
The way I'd describe the original intent was this - "Secured storage involves a username/password, so consumers can pass a username as this first param - then they get it back as part of the object in `getObject`."  Realizing that's pretty awkward (namely: you get that extra `username` prop that's not in your `Stored` type from `getObject) - this is how I'd characterize this PR: "The fact that storage includes a username & password is an implementation detail of using keychain for iOS.  This should not leak to SecureStore consumers, who just want to securely store/retrieve a typed object."